### DISCUSSION TITLE
Close a single Toast

### DISCUSSION THREAD

**Discussion by Tekkasoft** [Blazored/Toast#216] [Q&A]
Hi Team,
i'm using a renderfragment to show a custom toast. Inside are two buttons: one to accept and one to cancel. When I click on one of the two buttons, in addition to performing the desired action, I would also like to close the toast.
Calling the Clear* functions I don't find a ClearToast(...) but functions that close everything, by level or otherwise.
Could you help me understand how to do it, if it can be done?
In practice I should emulate the pressure on the x of a single toast, but I don't know how to do it.
waiting for your kind reply
congratulations for developing this component
and have a good day

**Comment by chrissainty** [comment]
There is an example of displaying a custom toast in the samples project you can reference.

**Reply by Tekkasoft** [reply]
Hi chrissainty

The problem is not the custom toast but the possibility of closing the single toast.
Is there any way to do this?
In the IToastService interface I don't see methods that allow to do this but maybe there is some way to do it.

Is it possible to use something like the following code:

var toast = ToastService.ShowError("..message..");
ToastService.HideToast(toast);

instead of

ToastService.ClearAll();

Greetings

**Reply by Tekkasoft** [reply]
I'm using 3.2.0 version
